src/main/java/com/github/vkorobkov/jfixtures/processor/Processor.java

```diff
@@ -40,11 +41,19 @@ private void processFixture(Fixture fixture) {
     }
 
     private void handleFixtureInstructions(Fixture fixture) {
-        List<Instruction> fixtureInstructions = new ArrayList<>();
         val table = context.getConfig().table(fixture.name);
+        List<Instruction> fixtureInstructions = new ArrayList<>();
+        List<String> beforeInsertInstructions = table.getBeforeInserts();
+
         if (CleanMethod.DELETE == table.getCleanMethod()) {
             fixtureInstructions.add(cleanupTable(fixture));
         }
+
+        if (!beforeInsertInstructions.isEmpty()) {
```

Comment:
We don't need this `if` - we can just pass empty stream/collection with the same result